### ISSUE TITLE
Help Needed To Understand Tensor Flow Weights and Gradient Histogram

### ISSUE THREAD

**Issue by ultrons** [tensorflow/tensorflow#454]
Hi,

I am experimenting with conv nets for sentiment analysis.
I used a basic convent architecture, my network looks like:
![screen shot 2015-12-09 at 10 58 08 am](https://cloud.githubusercontent.com/assets/15708499/11695403/cb3e4644-9e63-11e5-8bc2-c3d71baf1b2f.png)
I am trying to analyze the learning, I have a bad feeling about weights, Although I am early in the learning process, but the weights for 5x5 filter essentially suggest no updates/ update on only very few component , because histogram essentially remains the same.

Any hints on what I could be missing?

[TensorBoard_basic_convnet.pdf](https://github.com/tensorflow/tensorflow/files/57216/TensorBoard_basic_convnet.pdf)

Everyone has said that, none the less, I would say it too, Thanks for the amazing library.

Regards,
Vaibhav Singh

**Comment by girving** [comment]
This is a better question for stackoverflow, since it does not seem to be about a bug in tensorflow.  As a first guess, you may need to either wait longer or increase your learning rate, but it is hard to tell from so little information.

**Comment by ultrons** [comment]
Thanks Girving, I meant for it to be a more of how to question than a bug, Looking for if I interpret the histogram correctly or missing some basics. I will check on stackoverflow as well.

**Comment by girving** [comment]
To clarify: it's a good question, but we're trying to keep questions going to stackoverflow since it makes it easier for future people who hit the same problem to find the solution.
